src/thunk.ts

```diff
@@ -16,6 +16,7 @@ export interface ThunkFn {
 }
 
 function copyToThunk (vnode: VNode, thunk: VNode): void {
+  // mutate vnode.data because it's faster than using Object.assign later
```

Comment:
Which `Object.assign` is this referring to, please?

Reply:
this one https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign